rtt-target/src/init.rs

```diff
@@ -17,17 +17,17 @@ macro_rules! rtt_init_channels {
     (
         $field:expr;
         $number:literal: {
-            size: $size:literal
-            $( mode: $mode:ident )?
-            $( name: $name:literal )?
+            size: $size:expr
+            $(, mode: $mode:path )?
```

Comment:
It would be nice if the comma was at the end of each line and not at the start. and then trailing comma can be optional.

Reply:
I put the comma at the begin of each line, but it actually matches the comma at the end of last line, which can be read as "if there is another new parameter, then add a comma between 2 lines". In this way, the trailing comma is omitted (in fact it's not permitted to write the trailing comma)

Reply:
ah, if you mean commas at each line end, then it has to be there, the compiler ask me to put a comma after `$mode:path`

Reply:
That's okay :) I understand what the code does :) I added a new comment to clarify what I meant :)